### ISSUE TITLE
[WCAG] Poppers are not responsive

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3558]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Poppers are not repsonsive and can appear off-screen (require horizaontal scrolling) when viewed at high zoom layers. See the Notificationa panel for example.

Test by setting browser to 1280px width and zoomed to 400%

### Expected Behavior

Poppers should be responsive: Their width should adapt to the available viewport width w/o requireing horizontal scrolling.

Test all components that use <Popper>
